packages/sign/README.md

```diff
@@ -169,6 +169,7 @@ const bundler = new DSSEBundleBuilder({
 
 // Sign a thing
 const artifact = {
+  type: 'dsse_envelope',
```

Comment:
Let's update to make it clear that it should be some sort of content type
```suggestion
  type: 'text/plain',
```

Reply:
Sounds good